R/do_groupChromPeaks-functions.R

```diff
@@ -240,7 +241,7 @@ do_groupChromPeaks_density_par <- function(peaks, sampleGroups,
                                    sampleGroupTbl, minFr, minSmpls,
                                    sampleGroupNms, peakOrdr) {
         den <- density(z[, "rt"], bw = bw, from = rtr[1] - 3 * bw,
-                       to = rtr[2] + 3 * bw)
+                       to = rtr[2] + 3 * bw, n = max(512, 2^(ceiling(log2(diff(rtr)/(bw/2))))))
```

Comment:
yes. If rt/n is not smaller than the requested bw (how much smaller I don't know), the bw can never be achieved because the points in the density function will be more than bw apart.
This put a de-facto limit on how small the bw could be making separating some peaks impossible to separate.